cpu/cortexm_common/panic.c

```diff
@@ -16,14 +17,21 @@
  *
  * @author      Oliver Hahm <oliver.hahm@inria.fr>
  * @author      Joakim Nohlgård <joakim.nohlgard@eistec.se>
+ * @author      Toon Stegen <toon.stegen@altran.com>
  */
 
 #include "cpu.h"
 #include "lpm.h"
+#include "log.h"
+
+#ifdef DEVELHELP
+static void print_IPSR(void);
```

Comment:
I agree that would be cleaner. I did it this way because the coding conventions: 

> Every function needs a prototype in addition to its definition. If a prototype is specified within a .c file it has to be declared BEFORE any function definitions.
Is it ok to ignore the guideline for small functions like this?